picard/ui/mainwindow.py

```diff
@@ -377,6 +377,11 @@ def create_actions(self):
         self.add_directory_action.setShortcut(QtGui.QKeySequence(_("Ctrl+D")))
         self.add_directory_action.triggered.connect(self.add_directory)
 
+        if sys.platform == "darwin":
```

Comment:
Instead of multiple `sys.platform == "darwin"` tests, i think you should create a property `self.show_close_window` initialized in one place and test against it.

Reply:
Right, could also just check for `self.close_window_action`